CodeSniffer/Standards/Squiz/Sniffs/PHP/DiscouragedFunctionsSniff.php

```diff
@@ -41,9 +41,12 @@ class Squiz_Sniffs_PHP_DiscouragedFunctionsSniff extends Generic_Sniffs_PHP_Forb
      * @var array(string => string|null)
      */
     public $forbiddenFunctions = array(
-                                  'error_log' => null,
-                                  'print_r'   => null,
-                                  'var_dump'  => null,
+                                  'error_log'             => null,
+                                  'print_r'               => null,
+                                  'var_dump'              => null,
+                                  'var_export'            => null,
```

Comment:
These are used if you want to create custom error handler. So probably would end in error, when looking at Symfony's Components (VarDumper and Debug).